packages/babel-plugin-transform-modules-commonjs/src/index.ts

```diff
@@ -106,6 +107,7 @@ export default declare((api, options: Options) => {
 
     UpdateExpression(path) {
       const arg = path.get("argument");
+      if (!arg.isIdentifier()) return;
       const localName = arg.node.name;
       if (localName !== "module" && localName !== "exports") return;
```

Comment:
This code change doesn't have any effect on the result, other than making the code "sound". If `arg` is not an identifier, `arg.node.name` is undefined and thus we returned.